### ISSUE TITLE
BUG: Kubefile CMD not support Standard environment variable format

### ISSUE THREAD

**Issue by willzhang** [labring/sealos#2047]
### Sealos Version

v4.1.4-rc1

### How to reproduce the bug?

CMD env must be `$(CONT_IMG_VER)` 

```
FROM scratch

ENV CONT_IMG_VER=${CONT_IMG_VER:-v1.0.0}

CMD ["echo $(CONT_IMG_VER)"]
```

1、it let Ordinary users do not know this special situation
2、can not support `${CONT_IMG_VER:-v1.0.0}`

### What is the expected behavior?

CMD env should support `${CONT_IMG_VER}` or `$CONT_IMG_VER`

### Operating environment

```markdown
- Sealos version:
- Docker version:
- Kubernetes version:
- Operating system:
- Runtime environment:
- Cluster size:
- Additional information:
```

**Comment by willzhang** [comment]
```
root@node1:~# sealos run -f registry.cn-shenzhen.aliyuncs.com/cnmirror/env:v1.0  --env HELM_SET="externalURL=https://core.harbor.domain,core.image.repository=goharbor/harbor-core"
2022-11-05T21:57:12 info start to install app in this cluster
are you sure to override these app? Yes [y/yes], No [n/no] : y
2022-11-05T21:57:56 info pulling images [registry.cn-shenzhen.aliyuncs.com/cnmirror/env:v1.0] for platform linux/amd64
96cee5efb728e8afa76ff80c53d302d4b214d5c615ed49202d5f8ff117105665
default-xpa2lcg9
2022-11-05T21:57:57 info guest cmd is echo $(HELM_SET)
bash: line 1: HELM_SET: command not found

2022-11-05T21:57:57 info succeeded install app in this cluster
2022-11-05T21:57:57 info no nodes that need to be scaled
2022-11-05T21:57:57 info 
      ___           ___           ___           ___       ___           ___
     /\  \         /\  \         /\  \         /\__\     /\  \         /\  \
    /::\  \       /::\  \       /::\  \       /:/  /    /::\  \       /::\  \
   /:/\ \  \     /:/\:\  \     /:/\:\  \     /:/  /    /:/\:\  \     /:/\ \  \
  _\:\~\ \  \   /::\~\:\  \   /::\~\:\  \   /:/  /    /:/  \:\  \   _\:\~\ \  \
 /\ \:\ \ \__\ /:/\:\ \:\__\ /:/\:\ \:\__\ /:/__/    /:/__/ \:\__\ /\ \:\ \ \__\
 \:\ \:\ \/__/ \:\~\:\ \/__/ \/__\:\/:/  / \:\  \    \:\  \ /:/  / \:\ \:\ \/__/
  \:\ \:\__\    \:\ \:\__\        \::/  /   \:\  \    \:\  /:/  /   \:\ \:\__\
   \:\/:/  /     \:\ \/__/        /:/  /     \:\  \    \:\/:/  /     \:\/:/  /
    \::/  /       \:\__\         /:/  /       \:\__\    \::/  /       \::/  /
     \/__/         \/__/         \/__/         \/__/     \/__/         \/__/

                  Website :https://www.sealos.io/
                  Address :github.com/labring/sealos
                BuildVersion: 4.1.4-rc1-990b821d

root@node1:~# 
```

**Comment by willzhang** [comment]
is this fixed? i can use ${env} in sealos v4.1.5-alpine2

https://github.com/labring-actions/cluster-image/blob/main/applications/kube-vip/latest/Kubefile
